python312Packages.llama-index-core: 0.12.44 -> 0.12.46

fabaff opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [Nixpkgs 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/doc/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/doc/manual/release-notes/rl-2505.section.md) Nixpkgs Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
- [NixOS 25.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2511.section.md) (or backporting [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) NixOS Release notes)
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md), [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md), [maintainers/README.md](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md) and other contributing documentation in corresponding paths.

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python312Packages.llama-index-core: 0.12.44 -> 0.12.46

Diff: https://github.com/run-llama/llama_index/compare/refs/tags/v0.12.44...refs/tags/v0.12.46

Changelog: https://github.com/run-llama/llama_index/blob/v0.12.46/CHANGELOG.md

fabaff committed:
python312Packages.llama-parse: 0.6.37 -> 0.6.41

fabaff committed:
python312Packages.llama-index-vector-stores-qdrant: 0.6.0 -> 0.6.1

fabaff committed:
python313Packages.llama-index-agent-openai: 0.4.9 -> 0.4.12

fabaff committed:
python313Packages.llama-cloud: 0.1.29 -> 0.1.30

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 422665`
Commit: `e8eed44e5bb023d2e17c8ec7d29e4c6833822dcf`

---
### `x86_64-linux`
<details>
  <summary>:x: 14 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 135 packages built:</summary>
  <ul>
    <li>newelle</li>
    <li>python312Packages.llama-cloud</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-cloud.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python313Packages.llama-cloud</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-cloud.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
  </ul>
</details>

sarahec commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 422665`
Commit: `e8eed44e5bb023d2e17c8ec7d29e4c6833822dcf`

---
### `x86_64-linux`
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 143 packages built:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>newelle</li>
    <li>python312Packages.llama-cloud</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-cloud.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python313Packages.llama-cloud</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-cloud.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
  </ul>
</details>

sarahec reviewed (approved):
LGTM

fabaff committed:
python313Packages.llama-cloud: 0.1.30 -> 0.1.33

fabaff committed:
python313Packages.llama-cloud-services: 0.6.41 -> 0.6.46

Diff: https://github.com/run-llama/llama_cloud_services/compare/refs/tags/v0.6.41...refs/tags/v0.6.46

Changelog: https://github.com/run-llama/llama_cloud_services/releases/tag/v0.6.46

fabaff committed:
python313Packages.llama-index-cli: 0.4.3 -> 0.4.4

fabaff committed:
python313Packages.llama-index-indices-managed-llama-cloud: 0.7.7 -> 0.7.10

fabaff committed:
python313Packages.llama-index-readers-file: 0.4.9 -> 0.4.11

fabaff committed:
python313Packages.llama-index-workflows: 1.0.1 -> 1.1.0

fabaff committed:
python313Packages.llama-parse: 0.6.41 -> 0.6.46
